_posts/2020-09-01-weather-prediction/weather_prediction_deep_learning.Rmd

```diff
@@ -0,0 +1,693 @@
+
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanaweatherforecasting
+bibliography: bibliography.bib
+date: 09-01-2020
+categories:
+  - R
+  - TensorFlow/Keras
+  - Time Series
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/thumb.png
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+With all that is going on in the world these days, is it frivolous to talk about weather prediction? Asked in the 21st
+century, this is bound to be a rhetorical question. In the 1930s, when German poet Bertolt Brecht wrote the famous lines
+
+> Was sind das für Zeiten, wo\
+> Ein Gespräch über Bäume fast ein Verbrechen ist\
+> Weil es ein Schweigen über so viele Untaten einschließt![^1]
+
+[^1]: *An die Nachgeborenen, 1934-38.* The atrocities referred to are those of Nazi Germany.
+
+("What kind of times are these, where a conversation about trees is almost a crime, for it means silence about so many
+atrocities!"),
+
+he couldn't have anticipated the responses he would get in the second half of that century, with trees symbolizing, as well as
+literally falling victim to, environmental pollution and climate change.
+
+Today, no lengthy justification is needed as to why prediction of atmospheric states is vital: Due to global warming,
+frequency and intensity of severe weather conditions -- droughts, wildfires, hurricanes, heatwaves -- have risen and will
+continue to rise. And while accurate forecasts don't change those events per se, they constitute essential information in
+mitigating their consequences. This goes for atmospheric forecasts on all scales: from so-called "nowcasting" (operating on a
+range of about six hours) over medium-range (three to five days) and sub-seasonal (weekly/monthly) to climate forecasts
+(concerned with years and decades). Medium-range forecasts especially are extremely important in acute disaster prevention.
+
+This post will show how deep learning (DL) methods can be used to generate atmospheric forecasts, using a newly published
+[benchmark dataset](https://github.com/pangeo-data/WeatherBench)[@weatherbench]. Future posts may refine the model used here
+and/or discuss the role of DL ("AI") in mitigating climate change -- and its implications -- more globally. And coming back to
+Bertolt Brecht: While this text may be about trees, climate change should not remain the only societal topic addressed on this
+blog.
```

Comment:
I understand what you're saying here, but it feels a little out of place.

Reply:
You're so right of course ... I should just do it, not keep running around and telling everyone I will ;-)))